src/main/java/net/mcreator/ui/dialogs/wysiwyg/CheckboxDialog.java

```diff
@@ -51,20 +52,12 @@ public CheckboxDialog(WYSIWYGEditor editor, @Nullable Checkbox checkbox) {
 
 		VTextField nameField = new VTextField(20);
 		nameField.setPreferredSize(new Dimension(200, 28));
+		Validator validator = new UniqueNameValidator(nameField, "Checkbox machine name",

```

Comment:
L10N this, also for other one

Reply:
What should I use for the key?

Reply:
Check what other validators use and use something similar

Reply:
:heavy_check_mark: Fixed